.mailmap

```diff
@@ -49,9 +49,9 @@
 #   The mailmap file was reordered
 #
 #   For instructions on updating the .mailmap file see:
-#   https://docs.sympy.org/dev/contributing/new-contributors-guide/workflow-process.html#add-your-name-and-email-address-to-the-mailmap-file
+#   https://docs.sympy.org/dev/contributing/new-contributors-guide/workflow-.html#add-your-name-and-email-address-to-the-mailmap-file
 #
-#   ...
+#   ...process
```

Comment:
I don't understand these changes in the mailmap file. I am not sure we need them

Reply:
Oops i will fix it